daml-lf/kv-support/src/main/scala/com/daml/lf/kv/transactions/RawTransaction.scala

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.kv.transactions
+
+import com.google.protobuf.ByteString
+
+case class RawTransaction(byteString: ByteString) extends AnyVal
+
+object RawTransaction {
+  case class Node(byteString: ByteString) extends AnyVal
```

Comment:
where do you need to store nodes outside of a transaction?

Reply:
In pkvutils log fragments (oem integration kit).